docs/logs/log-rate.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[[xpack-logs-analysis]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+With the {ml} {anomaly-detect} features enabled,
+you can use the *Log rate* page in the Logs app to automatically detect and inspect log anomalies and
+the log partitions in which they occurred.
+This helps you to spot anomalous behavior without significant human intervention --
+no more manually sampling log data, calculating rates, and determining if rates are normal.
+
+*Log rate* automatically highlights periods of time where the log rate is outside expected bounds and therefore may be anomalous.
+You can use this information as a basis for further investigations.
+For example:
+* A significant drop in the log rate might suggest that some some piece of infrastructure has stopped responding,
+and thus we're serving less requests.
+* A spike in the log rate could denote a DDoS attack.
+This may lead to an investigation of IP addresses from incoming requests.
+
+TIP: You can also view log anomalies directly in the <<xpack-ml-anomalies,Machine Learning app>>.
+
+[float]
+[[logs-analysis-create-ml-job]]
+=== Create a machine learning job for log rate analysis
+
+To enable log rate analysis and anomaly detection,
+you must first create your own {kibana-ref}/xpack-spaces.html[space].
+Within a space, you can create a machine learning job which will carry out the log rate analysis.
+
+[role="screenshot"]
+image::logs/images/analysis-tab-create-ml-job.png[Create machine learning job]
+
+1. Select the time range for the machine learning analysis.
+By default, the analysis uses logs no older than four weeks, and continues indefinitely.
+You cannot change the time range for the analysis after the machine learning job has been created.
+2. Add the indices containing the logs you want to analyze.
+3. Click *Create ML job*.
+
+Even though the machine learning job's time range is fixed,
+you can still use the time filter to adjust the results that are shown in your analysis.
+
+[float]
+[[logs-analysis-view-log-entries]]
+=== View log entries
+
+The *Log rate* page displays the following:
+
+* <<logs-analysis-entries-chart>>
+* <<logs-analysis-anomalies-chart>>
+
+[role="screenshot"]
+image::logs/images/analysis-tab.png[Analysis tab in Logs app in Kibana]
+
+[float]
+[[logs-analysis-entries-chart]]
+==== Log entries chart
+
+The log entries chart shows an overall, color-coded visualization of the log entry rate,
+partitioned according to the value of the Elastic Common Schema (ECS)
+{ecs-ref}/ecs-event.html[`event.dataset`] field.
+
+
+If you have a lot of log partitions, the default view may not be the best way to examine your logs.
```

Comment:
`the default view may not be the best way to examine your logs.` is frustrating because I want to know the best way to examine my logs. Instead, how about:

If you have a lot of log partitions, use the following to filter your data:

* Hover over a time period (or range??) to see the log rate for each partition
* Click a partition name to show, hide, or highlight the partition values